Mithril update 2024-07-17 (#428)
IntersectMBO/cardano-updates: blog/2024-07-17-mithril.md

---
title: Mithril Team Update
slug: 2024-07-17-mithril
authors: jpraynaud
tags: [mithril]
hide_table_of_contents: false
---

## High level overview

> This week, the Mithril team released the new distribution [`2428.0`](https://github.com/input-output-hk/mithril/releases/tag/2428.0), which includes the support for Cardano node version `9.0.0`, as well as bug fixes and optimizations. They also continued implementing the certification of Cardano transactions in Mithril networks: they calibrated the signature parameters for mainnet, fixed a bug preventing the signature of the last block advertised as certified, optimized a part of the artifact production which was blocking the certification, and started removing the code reading transactions from the immutable files. 
> 
> Finally, the team started working on the certification of the Cardano stake distribution and kept redacting a draft CIP for the diffusion of Mithril signatures through the Cardano network.

Reviewer: ```suggestion
This week, the Mithril team released the new distribution [`2428.0`](https://github.com/input-output-hk/mithril/releases/tag/2428.0), which includes support for Cardano node version `9.0.0`, as well as bug fixes and optimizations. They also continued implementing the certification of Cardano transactions in Mithril networks by calibrating the signature parameters for mainnet, fixing a bug preventing the signature of the last block advertised as certified, optimizing a part of the artifact production that was blocking the certification, and starting removing the code reading transactions from the immutable files.

Finally, the team started working on the certification of the Cardano stake distribution and continued redacting a CIP for the diffusion of Mithril signatures through the Cardano network.
```
